src/I18n/DateFormatTrait.php

```diff
@@ -251,7 +251,7 @@ protected function _formatObject($date, $format, ?string $locale): string
                 $calendar,
                 $pattern
             );
-            if ($formatter === false) {
+            if (empty($formatter)) {
```

Comment:
psalm says return value of `datefmt_create()` can be false while phpstan says it can be `null`. It seems the PHP doc was updated at some point (or changed for PHP 8). Regardless, using `empty()` caters for both types.